TKGame/Components/Concrete/KinghtEnemy_PhysicsComponent.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TKGame.Components.Concrete;
+using TKGame.Components.Interface;
+using TKGame.BackEnd;
+using Myra.Graphics2D.UI;
+using System.Drawing;
+
+namespace TKGame.Components.Concrete
+{
+    class KinghtEnemy_PhysicsComponent : PhysicsComponent
+    {
+        void PhysicsComponent.Update(Entity entity, GameTime gameTime/*, World &world*/)
+        {
+            Player player = Player.Instance;
+            entity.Velocity.X = 1;
+            entity.Velocity.Y = 1;
```

Comment:
I'm not a huge fan of giving the enemy a static velocity when it could only be moving vertically (Xvel = 0 in reality) or the opposite with moving horizontally.